.github/workflows/tests.yml

```diff
@@ -20,28 +20,26 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        php: [ '5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1' ]
```

Comment:
If I remember correctly I haven't agreed to remove support for any of the PHP versions.

Please revert and keep PHP 7.x tool usage (which PHPStan is) locked to these PHP versions.